### PR TITLE
[android] initializing alohalytics on startup

### PR DESCRIPTION
**Pull request by milchakov**
Фикс креша на старте.
@ygorshenin 
@yunikkk

### CODE REVIEW

File: android/src/com/mapswithme/maps/MwmApplication.java [hunk 1]
```diff
@@ -107,6 +107,9 @@ public void onCreate()
     initHockeyApp();
     initCrashlytics();
     initPushWoosh();
+    // Workaround to initialize alohalytics for transferring http requests.
+    // Http wrapper should be cut out from the alohalytics in the future.
+    Statistics s = Statistics.INSTANCE;
```
Comment (ygorshenin) [inline android/src/com/mapswithme/maps/MwmApplication.java]: OK as a workaround. But need to extract http code into an individual unit and initialize it separately.

**Comment by ygorshenin** [comment]
LGTM, but please, move out http code from alohalytics.

File: android/src/com/mapswithme/maps/MwmApplication.java (hunk 1 above)
Comment (yunikkk) [inline android/src/com/mapswithme/maps/MwmApplication.java]: +100500 for extracting http module.

File: android/src/com/mapswithme/maps/MwmApplication.java [hunk 2]
```diff
@@ -107,6 +107,9 @@ public void onCreate()
     initHockeyApp();
     initCrashlytics();
     initPushWoosh();
+    // Workaround to initialize alohalytics for transferring http requests.
```
Comment (yunikkk) [inline android/src/com/mapswithme/maps/MwmApplication.java]: Leave TODO here, it will help to remember about the case.

**Comment by yunikkk** [comment]
LGTM, with or without TODO

**Comment by milchakov** [comment]
Я буду выпиливать сразу после своей текущей задачи, вмержите плиз

**Pull request merged**